ILSpy/Util/ShellHelper.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace ICSharpCode.ILSpy.Util
+{
+	static class ShellHelper
```

Comment:
@sonyps5201314 why not just keep Process.Start("explorer.exe", ....)? We'd prefer to not introduce more P/Invoke dependencies.